files-community/Files
# Feature: Git Integration - Phase 4

**Resolved / Related Issues**
   Related #11014

**Validation**
How did you test these changes?
   1. Open app
   2. Navigate to a git directory
   3. Pull changes

**Screenshots (optional)**
![image](https://github.com/files-community/Files/assets/102259289/6bdb1ea7-e530-40d8-92c1-1a2851e807c8)
